src/views/RecIM/components/List/Listing/index.js

```diff
@@ -110,24 +109,24 @@ const ParticipantListing = ({ participant }) => {
             variant="rounded"
           ></Avatar>
         </ListItemAvatar>
-        <Link to={`/profile/${participant.username}`} className="gc360_link">
-          <ListItemText primary={participant.username} />
+        <Link to={`/profile/${participant.Username.toLowerCase()}`} className="gc360_link">
```

Comment:
Why are you going tolowercase here? That has not been our practice in the past for 360 profiles. It is just extra code as far as functionality goes.